### ISSUE TITLE
Does the Growthbook iOS SDK support version strings for flag attributes?

### ISSUE THREAD

**Issue by geoff-vsco** [growthbook/growthbook#3091]
### Description of Feature

The latest version of the iOS SDK seems to be `1.0.62`.
https://github.com/growthbook/growthbook-swift/tags

Yet this [documentation](https://docs.growthbook.io/features/targeting) says: 

> Semantic Version Targeting
In version 2.2, we introduced support for semantic version string comparisons. Without this, the string 1.0.9 will be seen as "greater" than 1.0.10.

Since iOS SDK doesn't even go to v2.2...is there support for semantic version string comparisons for iOS?

**Comment by jdorn** [comment]
Yes — semantic version targeting is supported in growthbook-swift, and has been since mid-2023 (`Utils.paddedVersionString` backs all six version operators in the condition evaluator).

The "version 2.2" in the targeting docs refers to the GrowthBook app version, not the SDK version — SDK versioning is independent per language. Closing.
